### PR TITLE
web resize textarea on notebook tab show

### PR DESCRIPTION
**Pull request by msh-odoo**
Task:https://www.odoo.com/web?debug#id=2007558&action=333&active_id=1724&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.483fe0cc1077106f76e6ecce4a494d4e

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/tests/views/form_tests.js [hunk 1]
```diff
@@ -7016,6 +7016,52 @@ QUnit.module('Views', {
...
+        var height = form.$('.o_field_widget[name=text_field]').height();
+        assert.ok(height > 80, "textarea should have an height of at least 80px");
+
+        assert.containsN(form, 'textarea', 2,
+            "should have two textarea one for field and one fixed textarea");
+        // 2-2 px top-bottom padding in o_input element i.e. 0th textarea, check scrollHeight of
+        // textarea is equal to fixedTextarea created by dom.autoresize function
+        assert.strictEqual(form.$('textarea:eq(0)')[0].scrollHeight + 'px',
+                form.$('textarea:eq(1)')[0].scrollHeight + 4 + 'px',
+                "autoresized textarea should have height of fixed textarea + padding (0 line)");
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form_tests.js]: the last two asserts should be removed I think (there rely on implementation details). I think that testing that the height is at least 80 is enough. However, i'd add another (empty) textarea somewhere else in the view and assert that it's height is 50, to ensure that the > 80 assert is relevant.

Comment (msh-odoo) [reply addons/web/static/tests/views/form_tests.js]: @aab-odoo Done 👍

File: addons/web/static/tests/views/form_tests.js [hunk 2]
```diff
@@ -7016,6 +7016,55 @@ QUnit.module('Views', {
...
+        assert.hasClass(form.$('.o_notebook .nav .nav-link:nth(1)'), 'active');
+
+        var height = form.$('.o_field_widget[name=text_field]').height();
+        assert.ok(height > 80, "textarea should have an height of at least 80px");
+
+        await testUtils.dom.click(form.$('.o_notebook .nav .nav-link:nth(2)'));
+        assert.hasClass(form.$('.o_notebook .nav .nav-link:nth(2)'), 'active');
+
+        var height = form.$('.o_field_widget[name=text_field_empty]').css('height');
+        assert.strictEqual(height, '50px', "textarea should have an height of at least 50px");
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form_tests.js]: empty textarea should have an height of 50px

File: addons/web/static/tests/views/form_tests.js [hunk 3]
```diff
@@ -7016,6 +7016,55 @@ QUnit.module('Views', {
...
+
+
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form_tests.js]: one blank line is enough

File: addons/web/static/tests/views/form_tests.js (hunk 2 above)
Comment (msh-odoo) [reply addons/web/static/tests/views/form_tests.js]: 👍

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**